datastore/google/cloud/datastore_v1/gapic/datastore_client.py

```diff
@@ -464,9 +461,9 @@ def begin_transaction(
     def commit(
         self,
         project_id,
-        mode,
-        mutations,
```

Comment:
This is potentially breaking (required -> optional param) @frankyn @crwilcox could you check to see if this change looks intentional on the proto side? In some instances this has happened on accident (annotations being forgotten)